### PR TITLE
Update the content of /security/esm to match the copy doc

### LINKED ISSUE #10407: Copy updates on /security/esm for 20 September 2021
Please update /security/esm with the changes outlined in the [copy doc](https://docs.google.com/document/d/1RBZX3PJPvODxRlcnIS10Ii4SkIMH5Fm74EmsMQ0cnic/edit#heading=h.6zfiuksje0n5)

---
*Reported from: https://ubuntu.com/security/esm*

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Minor changes to match the copy doc
- Added a new column in the section "What is covered?"
- Added a new Q & A: "How to enable ESM?"

## QA

- Open the website: https://ubuntu-com-10410.demos.haus/security/esm
- Make sure that the changes matches the copy doc (link in the issue)
- Verify the newly added Q & A:  "How to enable ESM?" (with the code block)

## Issue / Card

Fixes #10407

### CODE REVIEW

File: templates/security/esm.html [hunk 1]
```diff
@@ -127,8 +127,8 @@ <h2>Free for personal use</h2>
 
     <div class="row" id="benefits">
       <div class="col-10">
-        <h2>What's covered?</h2>
-        <p>ESM continues security updates for high and critical CVEs (Common Vulnerabilities and Exposures).</p>
+        <h2>What is covered?</h2>
+        <p>ESM continues security updates <a href="/security/livepatch">and kernel livepatching</a> for high and critical CVEs (Common Vulnerabilities and Exposures).</p>
```
Comment (bethcollins92) [inline templates/security/esm.html:131]:
```suggestion
        <p>ESM continues security updates and<a href="/security/livepatch">kernel livepatching</a> for high and critical CVEs (Common Vulnerabilities and Exposures).</p>
```

File: templates/security/esm.html [hunk 2]
```diff
@@ -145,18 +145,26 @@ <h2>What's covered?</h2>
                 <td>
                   <ul>
                     <li>The Ubuntu base OS and scale-out infrastructure</li>
+                    <li>Kernel livepatching</li>
                     <li>64-bit x86 architecture</li>
                   </ul>
                 </td>
+                <td>
+                  <a href="https://wiki.ubuntu.com/SecurityTeam/ESM/14.04" class="p-button--positive">Learn more</a>
```
Comment (bethcollins92) [inline templates/security/esm.html:153]: Could you change the buttons to `p-button--neutral` please?

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good so far just a couple of changes. @long-chung could we get a design review please? I feel like the table with the learn more buttons would look better a bit like:
Screenshot 2021-09-20 at 15 31 29
or maybe even in the same column but underneath the bullet points - what do you think?

**Comment by long-chung** [comment]
@bethcollins92 I think having the CTA below the bullets would be better. Looking at the live site, the buttons are too close to the text and feels like its more related to the first bullet of each section

**Comment by goulinkh** [comment]
Thanks, I will push a version with buttons bellow the bullet list to see the result!

**Review by bethcollins92 — APPROVED** [review]
LGTM - thanks for updating

**Comment by bethcollins92** [comment]
@nmav could you QA this please? and give a Pdm +1 if it's good to go, thanks!

**Pull request merged**